libsolidity/lsp/LanguageServer.cpp

```diff
@@ -171,14 +172,48 @@ void LanguageServer::changeConfiguration(Json::Value const& _settings)
 		if (typeFailureCount)
 			m_client.trace("Invalid JSON configuration passed. \"include-paths\" must be an array of strings.");
 	}
+
+	Json::Value jsonRemappings = _settings["remappings"];
+
+	if (jsonRemappings)
+	{
+		int typeFailureCount = 0;
+		if (jsonRemappings.isArray())
+		{
+			vector<frontend::ImportRemapper::Remapping> remappings;
+			for (Json::Value const& jsonPath: jsonRemappings)
+			{
+				if (jsonPath.isObject())
+				{
+					frontend::ImportRemapper::Remapping remapping = {
+						jsonPath.get("context", "").asString(),
+						jsonPath.get("prefix", "").asString(),
+						jsonPath.get("target", "").asString(),
+					};
+					remappings.emplace_back(remapping);
```

Comment:
```suggestion
					remappings.emplace_back(ImportRemapper::Remapping{
						jsonPath.get("context", "").asString(),
						jsonPath.get("prefix", "").asString(),
						jsonPath.get("target", "").asString(),
					});
```

`frontend` namespace was imported on top of the file already and  constructing the parameter directly as part of the emplace-call will not make you accidentally forget to "move" the temporary.